Add `osu!taiko World Cup 2025: Finals Recap` news post (#13178)
ppy/osu-wiki: news/2025/2025-04-24-osutaiko-world-cup-finals-recap.md

---
layout: post
title: "osu!taiko World Cup 2025: Finals Recap"
date: 2025-04-24 20:15:00 +0000
---

The last remaining teams approached the summit. One final step to the ultimate battle!

![](/wiki/shared/news/2025-04-24-osutaiko-world-cup-finals-recap/banner.jpg)

> With only one stage left, every matchup was a high-stakes clash, with players conquering some of the most bural maps of the tournament so far. The skill cap climbed even higher as teams fought for their place in the top four. Relentless pressure and the Grand Finals within reach, the few players that were left did everything they could to save their spot on the podium.

Reviewer: ```suggestion
With only one stage left, every matchup was a high-stakes clash, with players conquering some of the most brutal maps of the tournament so far. The skill cap climbed even higher as teams fought for their place in the top four. Relentless pressure and the Grand Finals within reach, the few players that were left did everything they could to save their spot on the podium.
```

> The stage is now set for the Grand Finals, read along to find out how we got here!

Reviewer: ```suggestion
The stage is now set for the Grand Finals. Read along to find out how we got here!
```

## Mappool showcase

<style>
    .osu-md--news .audio-player {
        max-width: inherit;
    }
</style>

**[Download the mappool here! (78 MB)](https://packs.ppy.sh/P282%20-%20osu!taiko%20World%20Cup%202025%3A%20Finals.zip)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=2h4m42s&video=2425712130" allowfullscreen="true" scrolling="no"></iframe>
</div>

![Quarterfinals mappool table](/wiki/shared/news/2025-04-24-osutaiko-world-cup-finals-recap/mappool.png)

> Alongside the [newly announced featured artist](https://osu.ppy.sh/home/news/2025-04-13-new-featured-artist-terminal-11), we were treated to a brand new [osu! original](/wiki/Community/Bespoke_music) for this year's osu!taiko World Cup! [Terminal 11](https://osu.ppy.sh/beatmaps/artists/470) delivered **Echoes** which resonated throughout the Finals.

Reviewer: this is capitalised everywhere

```suggestion
Alongside the [newly announced Featured Artist](https://osu.ppy.sh/home/news/2025-04-13-new-featured-artist-terminal-11), we were treated to a brand-new [osu! original](/wiki/Community/Bespoke_music) for this year's osu!taiko World Cup! [Terminal 11](https://osu.ppy.sh/beatmaps/artists/470) delivered **Echoes** which resonated throughout the Finals.
```

Please click the banner below to check out the map, and watch the showcase video of [**Terminal 11 - Echoes**](https://youtu.be/P0vgkkz0DTs). Enjoy!

[![](/wiki/shared/news/2025-04-24-osutaiko-world-cup-finals-recap/echoes.jpg)](https://osu.ppy.sh/beatmapsets/2354615#taiko/5069524)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/470/Echoes/Terminal 11 - Echoes.mp3">
    </audio>
</div>

## Finals results

Here are all the outcomes laid out:

| Team A |  |  | Team B |
| --: | :-: | :-: | :-- |
| **Taiwan** ::{ flag=TW }:: | **7** | 3 | ::{ flag=CN }:: China |
| **Chile** ::{ flag=CL }:: | **7** | 3 | ::{ flag=GB }:: United Kingdom |
| Chile ::{ flag=CL }:: | 3 | **7** | ::{ flag=TW }:: **Taiwan** |
| **Japan** ::{ flag=JP }:: | **7** | 2 | ::{ flag=KR }:: South Korea |

> We unfortunately had to see a few teams leave the tournament once again. But we want to take a moment to thank the following teams for participating and congratulate them to get this far: ::{ flag=RU }:: Russia, ::{ flag=MY }:: Malaysia, ::{ flag=FR }:: France, ::{ flag=TR }:: Türkiye, ::{ flag=US }:: United States and ::{ flag=SG }:: Singapore.

Reviewer: ```suggestion
We unfortunately had to see a few teams leave the tournament once again, but we want to take a moment to thank the following teams for participating and congratulate them for getting this far: ::{ flag=RU }:: Russia, ::{ flag=MY }:: Malaysia, ::{ flag=FR }:: France, ::{ flag=TR }:: Türkiye, ::{ flag=US }:: United States and ::{ flag=SG }:: Singapore.
```

> Another week, another set of statistics for you to check out.  [This week's statistics spreadsheet](https://docs.google.com/spreadsheets/d/1cIzLO8If8qgSQ6vf8vEeVBDdHO6XTyZLEtNtqqHdXF8) holds all the information you need to know!

Reviewer: ```suggestion
Another week, another set of statistics for you to check out. [This week's statistics spreadsheet](https://docs.google.com/spreadsheets/d/1cIzLO8If8qgSQ6vf8vEeVBDdHO6XTyZLEtNtqqHdXF8) holds all the information you ever want to know!
```

## Match highlights

### ::{ flag=CL }:: Chile vs. ::{ flag=TW }:: Taiwan

**[View the match information here!](https://osu.ppy.sh/community/matches/117849019)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=0h3m35s&video=2437676743" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> With the two teams in the winner's bracket having already secured their podium placements, only one precious spot in Grand Finals was left for the 4 lower bracket lineups to fight over. After Saturday's matches, it came down to a climactic match-up between the powerhouses of Chile (#6) and Taiwan (#5), fresh off confident 7-3 victories over the United Kingdom and China respectively. The stakes were simple: win, and you reach the osu!taiko World Cup 2025 podium.

Reviewer: there are a few common ways to write it but we use "winners' bracket"

```suggestion
With the two teams in the winners' bracket having already secured their podium placements, only one precious spot in the Grand Finals was left for the 4 lower-bracket lineups to fight over. After Saturday's matches, it came down to a climactic match-up between the powerhouses of Chile (#6) and Taiwan (#5), fresh off confident 7–3 victories over the United Kingdom and China respectively. The stakes were simple: win, and you reach the osu!taiko World Cup 2025 podium.
```

> Taiwan's ban was interestingly [BilliumMoto - Sentinel](https://osu.ppy.sh/beatmapsets/2354654#taiko/5069626) (HR2) despite their strong history on the fast HR slot, likely a result of Chile's intimidating 3.06 million team score in their previous match; Chile responded by banning [Ludicin - Signal](https://osu.ppy.sh/beatmapsets/2354876#taiko/5070177) (NM6) into the very tech-oriented Taiwan squad, followed by a pick of [cosMo@Bousou-P feat. HARU - LowSpeed Fake Lyric](https://osu.ppy.sh/beatmapsets/2354609#taiko/5069510) (FM2)… that led to a quick break point from Taiwan! ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669)'s 99.6% HD full combo on the brutal antimod wasn’t enough to hold the pick, but ::{ flag=CL }:: [chibamasu](https://osu.ppy.sh/users/16067522)'s 99.48% run on [TOMOYU & Supire feat. RIHINA - greenREgeneration](https://osu.ppy.sh/beatmapsets/2354626#taiko/5069549) (HD1) stood tall above the rest to put Chile on the board with a counter break on Taiwan's first pick.

Reviewer: ```suggestion
Taiwan's ban was interestingly [BilliumMoto - Sentinel](https://osu.ppy.sh/beatmapsets/2354654#taiko/5069626) (HR2) despite their strong history on the fast HR slot, likely a result of Chile's intimidating 3.06 million team score in their previous match. Chile responded by banning [Ludicin - Signal](https://osu.ppy.sh/beatmapsets/2354876#taiko/5070177) (NM6) into the very tech-oriented Taiwan squad, followed by a pick of [cosMo@Bousou-P feat. HARU - LowSpeed Fake Lyric](https://osu.ppy.sh/beatmapsets/2354609#taiko/5069510) (FM2)... that led to a quick break point from Taiwan! ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669)'s 99.6% HD full combo on the brutal antimod wasn't enough to hold the pick, but ::{ flag=CL }:: [chibamasu](https://osu.ppy.sh/users/16067522)'s 99.48% run on [TOMOYU & Supire feat. RIHINA - greenREgeneration](https://osu.ppy.sh/beatmapsets/2354626#taiko/5069549) (HD1) stood tall above the rest to put Chile on the board with a counter break on Taiwan's first pick.
```

> After staving off a huge second half comeback to take Chile's second pick of [cosMo@BousouP feat. Hatsune Miku - DeroDeroDeroDe](https://osu.ppy.sh/beatmapsets/2317287#taiko/4962143) (HD2), Taiwan finally broke the break streak with a dominant win on [hugepulse - Kaitei Gensou Toshi](https://osu.ppy.sh/beatmapsets/1955875#taiko/5069532) (NM7), with ::{ flag=TW }:: [HarryAda](https://osu.ppy.sh/users/12892087) narrowly missing out on a 99.5% full combo that would give the winner’s bracket a run for their money! Chile held on with three amazing HDHR scores on [Umeboshi Chazuke - Galaxy in Toybox](https://osu.ppy.sh/beatmapsets/559156#taiko/1182564) (FM1) to make it 2-3, Taiwan printed a point from the polyrhythmic [KERO - Color Printer](https://osu.ppy.sh/beatmapsets/2167213#taiko/4572334) (HR1), and Chile kept up on [KNOWER - Time Traveler](https://osu.ppy.sh/beatmapsets/2354840#taiko/5070071) (DT1) by way of not one but three full-combo near misses on one of the week's toughest maps.

Reviewer: ```suggestion
After staving off a huge second half comeback to take Chile's second pick of [cosMo@BousouP feat. Hatsune Miku - DeroDeroDeroDe](https://osu.ppy.sh/beatmapsets/2317287#taiko/4962143) (HD2), Taiwan finally broke the break streak with a dominant win on [hugepulse - Kaitei Gensou Toshi](https://osu.ppy.sh/beatmapsets/1955875#taiko/5069532) (NM7), with ::{ flag=TW }:: [HarryAda](https://osu.ppy.sh/users/12892087) narrowly missing out on a 99.5% full combo that would give the winners' bracket a run for their money! Chile held on with three amazing HDHR scores on [Umeboshi Chazuke - Galaxy in Toybox](https://osu.ppy.sh/beatmapsets/559156#taiko/1182564) (FM1) to make it 2–3, Taiwan printed a point from the polyrhythmic [KERO - Color Printer](https://osu.ppy.sh/beatmapsets/2167213#taiko/4572334) (HR1), and Chile kept up on [KNOWER - Time Traveler](https://osu.ppy.sh/beatmapsets/2354840#taiko/5070071) (DT1) by way of not one but three full combo near-misses on one of the week's toughest maps.
```

> Now up by a score of 3-4, Taiwan seemingly found an extra gear that Chile couldn't quite seem to match. Two full combos on [Spire of Lazarus - Big Hat Logan](https://osu.ppy.sh/beatmapsets/2354620#taiko/5069531) (NM5) with an unreal 3x100 from ::{ flag=TW }:: [HarryAda](https://osu.ppy.sh/users/12892087), three scores with one combined miss on the highly stressful [Amanogawa Converter - Never play me](https://osu.ppy.sh/beatmapsets/2354618#taiko/5069528) (NM4) for one more break point, and one tightly contested, high-accuracy battle on [Rahatt - Gunoi Chop](https://osu.ppy.sh/beatmapsets/2354610#taiko/5069511) (NM1) later, team Taiwan had done it.

Reviewer: ```suggestion
Now up by a score of 3–4, Taiwan seemingly found an extra gear that Chile couldn't quite seem to match. Two full combos on [Spire of Lazarus - Big Hat Logan](https://osu.ppy.sh/beatmapsets/2354620#taiko/5069531) (NM5) with an unreal 3x100 from ::{ flag=TW }:: [HarryAda](https://osu.ppy.sh/users/12892087), three scores with one combined miss on the highly stressful [Amanogawa Converter - Never play me](https://osu.ppy.sh/beatmapsets/2354618#taiko/5069528) (NM4) for one more break point, and one tightly contested, high-accuracy battle on [Rahatt - Gunoi Chop](https://osu.ppy.sh/beatmapsets/2354610#taiko/5069511) (NM1) later, team Taiwan had done it.
```

> With a huge 7-3 win over a terrifying opponent in Chile, who still deserve huge congratulations for their best finish ever at 4th place, Taiwan came out on top of the brutal Finals lower bracket, advancing to the Grand Finals.

Reviewer: "who" is generally for people
"Chile deserve" is also weird because the country is grammatically not plural

```suggestion
With a huge 7–3 win over a terrifying opponent in Chile, a team that still deserves huge congratulations for their best finish ever at 4th place, Taiwan came out on top of the brutal Finals lower bracket, advancing to the Grand Finals.
```

## Score highlights

> In a later stage like the Finals, there is no avoiding scores that seem absoultely inhumane to a spectator. For players that have never played osu!taiko themselves, and even for players that are longstanding members of the community, some of those scores are incomprehensible.

Reviewer: ```suggestion
In a later stage like the Finals, there is no avoiding scores that seem absolutely inhumane to a spectator. For players who have never played osu!taiko themselves, and even for players who are longstanding members of the community, some of those scores are incomprehensible.
```

> - One of those scores last weekend, was the unreal double S score from ::{ flag=JP }:: [_Kan2](https://osu.ppy.sh/users/7160196) on the very experimental and difficult NM2 pick, [Terminal 11 - Echoes](https://osu.ppy.sh/beatmapsets/2354615#taiko/5069524). The Japanese player has outscored every other player in their match against South Korea with zero 100s and outstanding gameplay throughout our brand new osu! original.

Reviewer: the last bit sounds extra when the focus is otherwise on the score

```suggestion
- One of those scores last weekend was the unreal double S score from ::{ flag=JP }:: [_Kan2](https://osu.ppy.sh/users/7160196) on the very experimental and difficult NM2 pick, [Terminal 11 - Echoes](https://osu.ppy.sh/beatmapsets/2354615#taiko/5069524). The Japanese player has outscored every other player in their match against South Korea with zero 100s and outstanding gameplay.
```

> - It takes a lot of confidence and practice to pick any mod on an antimod pick. But to go all the way picking **two** mods on it and scoring a full combo on said pick, deserves a mention in this list. This is exactly what ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) and ::{ flag=JP }:: [Maimaing](https://osu.ppy.sh/users/14520910) did. Both of these players have not missed a single note throughout [cosMo@Bousou-P feat. HARU - LowSpeed Fake Lyric](https://osu.ppy.sh/beatmapsets/2354609#taiko/5069510) (FM2) and thanks to the godlike accuracy of 99.60% on ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027)'s side, South Korea won the pick.

Reviewer: ```suggestion
- It takes a lot of confidence and practice to pick any mod on an antimod pick, but to go all the way with **two** mods and scoring a full combo despite this... definitely deserves a mention. This is exactly what ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) and ::{ flag=JP }:: [Maimaing](https://osu.ppy.sh/users/14520910) did. Both of these players have not missed a single note in [cosMo@Bousou-P feat. HARU - LowSpeed Fake Lyric](https://osu.ppy.sh/beatmapsets/2354609#taiko/5069510) (FM2), and thanks to the godlike accuracy of 99.60% on ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027)'s side, South Korea won the pick.
```
